app/src/main/java/com/example/concordia_campus_guide/Database/Converters/DoubleListConverter.java

```diff
@@ -1,28 +0,0 @@
-package com.example.concordia_campus_guide.Database.Converters;
-
-
-import androidx.room.TypeConverter;
```

Comment:
I thought we still used this converter, is this not the case?

Reply:
This converter was used to convert the double list of the centerCoordinates from the place class. since u refactored and changed it to use the Coordinates class, this converter wasn't used anymore.